edx_reverification_block/xblock/reverification_block.py

```diff
@@ -321,8 +321,7 @@ def validate(self):
                 ValidationMessage(
                     ValidationMessage.WARNING,
                     self._(
-                        u"This checkpoint is not associated with an assessment yet. "
-                        u"Please select an Assessment."
+                        u"Select Edit to associate the checkpoint with an assessment."
```

Comment:
Update based on feedback about the validation message:

This checkpoint is not associated with an assessment. To associate the checkpoint with an assessment, select Edit.